src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -275,7 +275,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _unfocusedAppearanceViewModel.Schemes(schemes);
         _unfocusedAppearanceViewModel.WindowRoot(windowRoot);
 
-        _NotifyChanges(L"UnfocusedAppearance", L"HasUnfocusedAppearance");
+        _NotifyChanges(L"UnfocusedAppearance", L"HasUnfocusedAppearance", L"ShowUnfocusedAppearance");
```

Comment:
Why does this one notify a change for "UnfocusedAppearance" but `DeleteUnfocusedAppearance()` doesn't? 🤔

Reply:
It's a fair question. If somebody were to be monitoring that object, it would have changed. We may want to notify on it just to be safe!

Reply:
Mostly because it just wasn't necessary since XAML hides the entire control once it gets the `HasUnfocusedAppearance`/`ShowUnfocusedAppearance` update, but yeah I've added it just in case!